VideoLocker/src/main/java/org/edx/mobile/authentication/LoginAPI.java

```diff
@@ -161,38 +166,38 @@ public AuthResponse registerUsingFacebook(@NonNull Bundle parameters, @NonNull S
     }
 
     @NonNull
-    public ResetPasswordResponse resetPassword(@NonNull String email) throws HttpException {
-        return loginService.resetPassword(email);
-    }
-
-    @NonNull
     private void register(Bundle parameters) throws Exception {
```

Comment:
How difficult would it be to throw a more specific exception? I know you didnt change the method signature in the effort, but the body did change a lot.  Maybe throws HttpResponseStatusException, RegistrationException?